### DISCUSSION TITLE
Public Wado Service

### DISCUSSION THREAD

**Discussion by esefberfu** [dcm4che/dcm4chee-arc-light#2975] [Q&A] [answered]
I have installed dcm4chee arc light on docker.
Ref: https://github.com/dcm4che/dcm4chee-arc-light/wiki/Run-secured-archive-services-on-a-single-host

I need to public wado service without any auth mechanism or only using url auth.
How can i do that ?

**Comment by gunterze** [comment, marked as answer]
Replace image `dcm4che/dcm4chee-arc-psql:5.23.0-secure` by  `dcm4che/dcm4chee-arc-psql:5.23.0-secure-ui`.

**Note:** Don't forget to purge mapped out `/opt/wildfly/standalone/deployments` and `/opt/wildfly/standalone/configuration`, to force updating the deployed EAR from the changed image.
